groups/instance/security_groups.tf

```diff
@@ -8,15 +8,15 @@ resource "aws_security_group" "instance_security_group" {
     from_port        = 8081
     to_port          = 8081
     protocol         = "tcp"
-    cidr_blocks      = local.placement_subnet_cidrs
+    cidr_blocks      = data.aws_ec2_managed_prefix_list.administration
```

Comment:
This will now prevent access from the previously-defined `cidr_blocks`. For the web access, it would be good to concatenate these two lists together since web access will be required from the placement subnets, from the automation subnets and the administration subnets.

This could be done as a new local `instance_web_access_cidrs` or something along those lines which uses `concat()` to merge `local.placement_subnet_cidrs` and `data.aws_ec2_managed_prefix_list.administration`.